OMNA/wsp/wsp-header-parameters.html

```diff
@@ -71,7 +71,7 @@
         <!--DTDs starts-->
 
         <div class="fl size99of100 mtm mbs">
-            <p>The information to be included in a request may be found <a href="/OMNA/wsp/wsp-header-parameters.html">here</a>.</p>
+            <p>The information to be included in a request may be found <a href="http://openmobilealliance.github.io/OMNA/registerpage.html#wsp-header-parameters">here</a>.</p>

```

Comment:
Please use relative URLs when navigating within the same server domain (i.e. openmobilealliance.github.io).